packages/rx-utils/src/rd/entity-store.utils.ts

```diff
@@ -34,8 +33,8 @@ export class EntityStore<L = never, A = never> {
 	private isLoadingAll = false;
 	private _getAllValues$ = this.cache.values$.pipe(
```

Comment:
Why don't you use only one pipe(fp-ts)?

Reply:
If it is about "other refactoring" could you add issues, please?

Reply:
Unrelated for fp-ts@2 migration.

I'm not sure what issued do you suggest - "refactoring rxjs pipes to fp-ts pipes"? What for?

Reply:
This is question to you :)
"All other refactoring which is out of scope of migration should be done separately."
What is "All other refactoring"? And should we do this or is it unnecessary?

Reply:
`All other refactoring` means any refactoring _not_ related to effects of upgrades. This includes: 
 - codestyle
 - deprecations (this case)
 - performance
 - any other code changes

This PR is meant to include minimal changes to keep things working the same as before.